lib/src/reactive_ble.dart

```diff
@@ -220,14 +220,22 @@ class FlutterReactiveBle {
         .then((message) => message.result.dematerialize());
   }
 
-  /// Scan for devices that are advertising a specific service
-  /// [scanMode] is used only on Android to enforce a more battery or latency intensive scan strategy
-  ///
-  Stream<DiscoveredDevice> scanForDevices(
-      {@required Uuid withService, ScanMode scanMode = ScanMode.balanced}) {
+  /// Scan for devices that are advertising a specific services. Use [withServices] filter on devices that are advertising
+  /// the defined services. 
+  /// 
+  /// There are two Android specific parameters that will be ignored on iOS: 
+  /// [scanMode] is used to enforce a more battery or latency intensive scan strategy,
```

Comment:
Maybe "allows to choose between different levels of power efficient and/or low latency scan modes"?